.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,17 @@
+name: 🚀 Feature Request
+description: Submit a proposal/request for a new db-ally feature.
+title: "[Feature]: "
+labels: ["enhancement"]
+body:
```

Comment:
We can ask for more here too. Like:
- **Is your feature request related to a problem? Please describe.** - A clear and concise description of what the problem is, e.g., I'm always frustrated when [...]
- **Describe the solution you'd like** - A clear and concise description of what you want to happen.
- **Describe alternatives you've considered** - A clear and concise description of any alternative solutions or features you've considered.
- **Additional context** - Add any other context or screenshots about the feature request here.